packages/tasit-contracts/abi/LANDProxy.json

```diff
@@ -185,257 +183,4 @@
-  "source": "pragma solidity ^0.4.18;\n\nimport \"../Storage.sol\";\nimport \"../upgradable/Proxy.sol\";\n\n\ncontract LANDProxy is Storage, Proxy {\n}\n",
-  "sourcePath": "/home/cmarcelo/git/TasitSDK/packages/tasit-contracts/decentraland/land/contracts/upgradable/LANDProxy.sol",
-  "ast": {
-    "absolutePath": "/home/cmarcelo/git/TasitSDK/packages/tasit-contracts/decentraland/land/contracts/upgradable/LANDProxy.sol",
-    "exportedSymbols": {
-      "LANDProxy": [
```

Comment:
Why were so many lines removed from specifically this ABI?

Reply:
We are storing only the `abi` arrays of each contract, that file was contained the entire contract `json`.

Reply:
Got it. Thanks! Resolving